test/Winton.Extensions.Configuration.Consul.Test/Parsers/Json/JsonPrimitiveVisitorTests.cs

```diff
@@ -77,6 +79,21 @@ public sealed class VisitJObject : JsonPrimitiveVisitorTests
                 }
             };
 
+            [Fact]
+            private void ShouldConvertPrimitivesToStringUsingJsonSerializerCulture()
+            {
+                Thread.CurrentThread.CurrentCulture = new CultureInfo("de-DE");
```

Comment:
Does this affect anything later? Or revert?

Reply:
Good point, I will revert it to be on the safe side.